cypress/integration/login.js

```diff
@@ -20,5 +20,11 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('TestCon User 8')
         cy.get('.main-nav').find('li').should('have.length', 2)
+
+    
+    })
+
+    it('Shoud display the date as current date', function (){
+        cy.get('.calendar--selected').should('have.class','calendar--today')
```

Comment:
A good approach for testing that today date is also a selected date, but I would expect to assert that today is a correct one like today is 13th so I would expect today's date in application to be 13th as well